JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -0,0 +1,55 @@
+using Amazon;
+using JustSaying.Sample.Restaurant.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Threading.Tasks;
+
+namespace JustSaying.Sample.Restaurant.KitchenConsole
+{
+    internal class Program
+    {
+        private static async Task Main()
+        {
+            Console.Title = "KitchenConsole";
+
+            await new HostBuilder()
+               .ConfigureLogging((hostContext, logging) =>
+               {
+                   logging.AddConsole();
+               })
+              .ConfigureServices((hostContext, services) =>
+              {
+                  services.AddJustSaying(config =>
+                  {
+                      config.Client(x =>
+                      {
+                          // Use the simulated pafortin/goaws docker image
+                          x.WithServiceUrl("http://localhost:4100");
+
+                          // or supply credentials if using the real aws services
+                          //x.WithCredentials(...)
+                      });
+                      config.Messaging(x =>
+                      {
+                          x.WithRegion(RegionEndpoint.APSoutheast2);
+                      });
+                      config.Subscriptions(x =>
+                      {
+                          x.ForTopic<OrderPlacedEvent>();
+                      });
+                      config.Publications(x =>
+                      {
+                          x.WithTopic<OrderReadyEvent>();
+                      });
+                  });
+                  services.AddJustSayingHandler<OrderPlacedEvent, OrderPlacedEventHandler>();
+                  services.AddHostedService<Subscriber>();
+              })
+              .UseConsoleLifetime()
+              .Build()
+              .RunAsync().ConfigureAwait(false);
```

Comment:
No need for `ConfigureAwait(false)` in this project. Console apps and ASP.NET Core applications do not have a `SyncronizationContext`.

Reply:
I think this is more a reaction to the catch-all code analyser suggestions.

Reply:
I think this is more a reaction to the catch-all code analyser suggestions.

Reply:
It is - I've had to add a `SuppressMessage` in order to make it compile without `ConfigureAwait`